app/models/activity.rb

```diff
@@ -77,6 +78,10 @@ def is_programme_level?
     level == "programme"
   end
 
+  def is_project_level?
```

Comment:
I found out last week that we can replace these methods with `activity.project?` because we get it for "free" with the enum!! 🌟 🎩

Reply:
If you like that, you (may) love this! 

You can also set a custom prefix and suffix which means we can keep 'is_<level>_level?` which I think is nice and explicit and still not have to make the methods!

Reply:
WHAT MY MIND IS BLOWN